src/mwoffliner.lib.ts

```diff
@@ -187,6 +187,9 @@ async function execute(argv: any) {
     webp,
   });
 
+  /* perform login */
+  await mw.login(downloader);
```

Comment:
shifted this here as in many cases site info might be restricted as well without login and login should be done before making most basic requests

Reply:
Good